lib/generators/inertia_rails/install_generator.rb

```diff
@@ -32,6 +33,8 @@ def installable?
 
         return false
       end
+
+      true
```

Comment:
That's what we get for refactoring without tests 🙄 

Thanks for catching this!